### PR TITLE
Added touch screen

### PR DESCRIPTION
**Pull request by Tianster**

### CODE REVIEW

**Comment by Tianster** [comment]
@weejh review

**Comment by weejh** [comment]
review 2

File: game/index.html [hunk 1]
```diff
@@ -4,6 +4,7 @@
     <meta charset="utf-8">
     <title>Tic Tac Toe</title>
     <link rel="stylesheet" href="game.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
```
Comment (weejh) [inline game/index.html]: review 3

**Comment by weejh** [comment]
@cbas help

**Pull request merged**